force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +325,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        Integer result = 0;
         // Loop through the list of integers
-            // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        for(Integer posnum: numbers ){
+               // if the number is negative skip this iteration
+               if(posnum >0){
+                result = result + posnum;
+               }
+        }
```

Comment:
A minor detail but I would not name my loop variable posNum. You are looking for positiive numbers that does not mean the next member of the collection is one. It is easy to see that name and assume the value is a positive number. 

```
 public static Integer sumPositiveIntegers() {
        Integer sum = 0;
        List<Integer> numbers = new List<Integer>{
                -1, 2, -3, 4, -5, 6
        }; //DO NOT CHANGE
        for (Integer num : numbers) {
            if(num > 0){
                sum += num;
            }
        }
        return sum; // Replace null with the variable you used to store the result
    }
 ```